tests/kafkatest/tests/core/throttling_test.py

```diff
@@ -137,12 +141,13 @@ def reassign_partitions(self, bounce_brokers, throttle):
                 estimated_throttled_time,
                 time_taken))
 
-    @parametrize(bounce_brokers=False)
-    @parametrize(bounce_brokers=True)
-    def test_throttled_reassignment(self, bounce_brokers):
+    @parametrize(bounce_brokers=False, use_config_command=False)
+    @parametrize(bounce_brokers=False, use_config_command=True)
+    def test_throttled_reassignment(self, bounce_brokers, use_config_command):
         security_protocol = 'PLAINTEXT'
         self.kafka.security_protocol = security_protocol
         self.kafka.interbroker_security_protocol = security_protocol
+        self.use_config_command = use_config_command
```

Comment:
Wouldn't it be better to pass this through to `reassign_partitions` rather than making it a member variable?